PUGTeologia.csl

```diff
@@ -0,0 +1,509 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" initialize-with-hyphen="true" page-range-format="expanded" demote-non-dropping-particle="sort-only" name-delimiter=" – " names-delimiter=" – " delimiter-precedes-last="always" et-al-min="4" et-al-use-first="1" name-form="long" initialize-with=". " sort-separator=", ">
+  <info>
+    <title>PUGTeologia</title>
```

Comment:
Please spell out the whole institution like this:
`Full university name - Faculty or department name`